app/controllers/submit.js

```diff
@@ -333,8 +338,38 @@ export default Ember.Controller.extend(BasicsValidations, NodeActionsMixin, Tagg
             this.send('next', this.get('_names.0'));
         },
         resetFileUpload() {
-            this.send('changeState', State.START);
-            this.set('fileAndNodeLocked', false);
+            var promisesArray = [];
+            var filePromises = [];
+            this.get('filesUploadedForPreprint').forEach((file) => {
+                filePromises.push(this.get('fileManager').deleteFile(file).then(() => {
+                    this.get('toast').info('File removed!');
+                }).catch(() => {
+                    this.get('toast').error('Could not remove uploaded file');
+                }));
+            });
+
+            Ember.RSVP.all(filePromises).then(() => {
+                this.get('projectsCreatedForPreprint').forEach((project) => {
+                    promisesArray.push(this.store.adapterFor('node').ajax(project.get('links.self'), 'DELETE').then(() => {
```

Comment:
Can you please clarify the intent of this line vs using ember's native methods like `node.deleteRecord().save()` or `node.destroyRecord()`?

Are we intentionally circumventing the store here to avoid cleaning up related record items? Should we be?